docs/source/install.rst

```diff
@@ -7,19 +7,30 @@ The Docker image is about 350MB and can be started with:
 
 .. code-block:: bash
 
-    docker run -e DEMO=1 dedica/nivio
+    docker run dedica/nivio
 
 Demo mode
 ---------
 
-In the demo mode Nivio loads sample data for demonstration purposes. From the `./nivio-demo` directory, run
+.. code-block:: bash
+
+    docker run -e DEMO=1 dedica/nivio
+
+In the demo mode Nivio loads sample data for demonstration purposes.
+
+
+There is a demo in the directory `./nivio-demo` which starts an nginx to serve all example configs from the project and
+starts nivio as docker container.
+
+From the directory, run
```

Comment:
Alright, thanks. But now the `docker-compose.yml` does not have any environment variables set and when I run it, I see no landscapes in the UI. Do we still need to update the `docker-compose.yml`?